MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -388,21 +388,39 @@ open class RouteStepProgress: NSObject {
     /**
      The next intersection the user will travel through.
      
-     The step must contains `Intersections` for this value not be `nil`.
+     The step must contain `intersectionsIncludingUpcomingManeuverIntersection` for this value not be `nil`.
```

Comment:
This is still unclear, because `intersectionsIncludingUpcomingManeuverIntersection` is a property of RouteStepProgress, not RouteStep.

> If `intersectionsIncludingUpcomingManeuverIntersection` is `nil`, this property is also `nil`.